docs/modules/ROOT/pages/integrations/database.adoc

```diff
@@ -0,0 +1,114 @@
+= Database Integrations
+
+== Overview
+
+KubeArchive requires a database to store resources.
+This document list the databases supported, how to set up the database schema
+and how to add new database integrations.
+
+=== PostgreSQL
+
+The only currently full supported.
+KubeArchive integration test suite runs with PostgreSQL 16.6
+
+The PostgreSQL implementation is available
+link:https://github.com/kubearchive/kubearchive/blob/main/pkg/database/postgresql.go[here].
+
```

Comment:
The PostgreSQL schema is available

link:https://github.com/kubearchive/kubearchive/blob/main/integrations/database/postgresql/kubearchive.sql[here]
and
link:https://github.com/kubearchive/kubearchive/releases/latest[the latest release as `kubearchive.sql`].

Reply:
applied suggestion